test/e2e/kubectl/kubectl.go

```diff
@@ -2606,10 +2608,19 @@ func createObjValidateOutputAndCleanup(client dynamic.ResourceInterface, obj *un
 
 	splitOutput := strings.SplitN(output, "\n", 2)
 	fields := strings.Fields(splitOutput[0])
-	if resource.Namespaced {
-		framework.ExpectNotEqual(fields, []string{"NAMESPACE", "NAME", "CREATED", "AT"}, fmt.Sprintf("expected non-default fields for namespaced resource: %s", resource.Name))
-	} else {
-		framework.ExpectNotEqual(fields, []string{"NAME", "AGE"}, fmt.Sprintf("expected non-default fields for resource: %s", resource.Name))
```

Comment:
this was an incorrect check that confused client-side and server-side default rendering with namespaced/cluster-scoped rendering, and incorrectly identified some types (like roles) as having custom rendering